### DISCUSSION TITLE
Closed at this time

### DISCUSSION THREAD

**Discussion by AlexValue** [organicmaps/organicmaps#2725] [Ideas]
I suggest that when you click on a company, show under the name an inscription about the availability at the current time, based on the work schedule.

When the place is open, you can write in black "Open until 21:00" for example. At 20:00 you can write "Closes in 1 hour" in orange. Then "Closes in X minutes". And finally, when the place is closed, you can write in red "Closed until 8:00" or "Closed until Monday" for example.

**Comment by Kazing** [comment]
This suggestion is already referenced in this issue : #1852
